backend/Generator/main.py

```
@@ -22,6 +22,50 @@
+            model = model.half()
+        elif MODEL_PRECISION == 'int8' and device.type == 'cpu':
+            # Apply INT8 dynamic quantization for CPU
+            logger.info("Applying INT8 dynamic quantization for CPU inference")
+            model = torch.quantization.quantize_dynamic(
+                model,
+                {torch.nn.Linear},
+                dtype=torch.qint8
+            )
+        else:
+            logger.info(f"Quantization skipped: precision={MODEL_PRECISION}, device={device.type}")
+    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)